addons/web_editor/static/src/js/editor/odoo-editor/test/spec/color.test.js

```diff
@@ -20,9 +20,9 @@ describe('applyColor', () => {
     it('should apply a background color to a slice of text in a span in a font', async () => {
         await testEditor(BasicEditor, {
             contentBefore: '<p>a<font class="a">b<span class="b">c[def]g</span>h</font>i</p>',
-            stepFunction: setColor('rgb(255, 0, 0)', 'backgroundColor'),
+            stepFunction: setColor('#FF0000', 'backgroundColor', true),
```

Comment:
In editor when we apply `backColor` from the first `theme-color` section, `applyColor` gets the color in HEX form. If the color is applied with custom opacity from `custom-color` section, it gets color in rgba form. In applyColor method we apply opacity only if it is in HEX form (Because it is from theme-color secction). In order to add opacity further, I changed the form of color from rgb to HEX.